ironmon_tracker/Debug/Tracker-StreamerBot.cs

```diff
@@ -534,7 +534,15 @@ private void VerifyOrCreateDataFiles()
 			// Check first if the user has defined alternative data folder
 			var directoryOverride = CPH.GetGlobalVar<string>(GVAR_ConnectionDataFolder, true);
 			if (!string.IsNullOrEmpty(directoryOverride) && !directoryOverride.Equals("NONE") && Directory.Exists(directoryOverride))
-				dataDirectory = directoryOverride;
+			{
+				_inboundFile = Path.Combine(directoryOverride, INBOUND_FILENAME); // Responses (incoming)
+				_outboundFile = Path.Combine(directoryOverride, OUTBOUND_FILENAME); // Requests (outgoing)
+			}
+			else
+			{
+				_inboundFile = Path.Combine(dataDirectory, DATA_FOLDER, INBOUND_FILENAME); // Responses (incoming)
+				_outboundFile = Path.Combine(dataDirectory, DATA_FOLDER, OUTBOUND_FILENAME); // Requests (outgoing)
+			}
 
 			// Create required inbound/outbound files
 			_inboundFile = Path.Combine(dataDirectory, DATA_FOLDER, INBOUND_FILENAME); // Responses (incoming)
```

Comment:
the `if` block here gets entirely overridden by the subsequent setting of `_inboundFile` and `_outboundFile` on lines 548,549 below it

Reply:
Good catch. I forgot to remove those lines.